spec/02-integration/09-hybrid_mode/01-sync_spec.lua

```diff
@@ -732,6 +732,10 @@ describe("CP/DP config sync #" .. strategy .. " rpc_sync=" .. rpc_sync, function
         end
       end, 5)
 
+      if rpc_sync == "on" then
+        ngx.sleep(0.5)
+      end
+
```

Comment:
hi @chronolaw hope this not causing flakiness. and need to add a comment, i guess here we just wait for cp/dp to transfer capslist?

Reply:
added.

Reply:
we need to add a mechanism to allow the test case to know when the rpc sync finished, might a error log

Reply:
Could we create a ticket for this idea?